samples/nrf_desktop/configuration/nrf52840_pca20041/battery_def.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/* This configuration file is included only once from battery level measuring module
+ * and holds information about battery characteristic.
+ */
+
+/* This structure enforces the header file is included only once in the build.
+ * Violating this requirement triggers a multiple definition error at link time.
+ */
+const struct {} battery_def_include_once;
+
+/* Converting measured battery voltage[mV] to State of Charge[%].
+ * First element corresponds to CONFIG_DESKTOP_BATTERY_MIN_LEVEL.
+ * Each element is CONFIG_DESKTOP_VOLTAGE_TO_SOC_DELTA higher than previous.
+ * Defined separately for every configuration.
+ */
+static const u8_t battery_voltage_to_soc[] = {
```

Comment:
Is it the characteristic for Thingy battery or was it measured for the 3-pin battery that we now have?

Reply:
It was from Thingy battery, we may changed it later on to match hardware configuration